syntaxes/vscode-scl.tmLanguage.json

```diff
@@ -119,7 +119,7 @@
 		"support":{
 			"patterns": [
 				{
-					"match": "(?i)\\b(ANY|ARRAY|BOOL|BYTE|CHAR|COUNTER|DATE|DATE_AND_TIME|SINT|SINT|INT|DINT|LINT|DT|DWORD|POINTER|REAL|LREAL|S5TIME|STRING|TIME|TIMER|TIME_OF_DAY|TOD|USINT|UINT|UDINT|ULINT|VOID|WORD|WSTRING|TON_TIME|TOF_TIME|TP_TIME|IEC_TIMER)\\b",
+					"match": "(?i)\\b(ANY|ARRAY|BOOL|BYTE|CHAR|COUNTER|DATE|DATE_AND_TIME|SINT|SINT|INT|DINT|LINT|DTL|DT|DWORD|POINTER|REAL|LREAL|S5TIME|STRING|TIME|TIMER|TIME_OF_DAY|TOD|USINT|UINT|UDINT|ULINT|VOID|WORD|WSTRING|TON_TIME|TOF_TIME|TP_TIME|IEC_TIMER|VAL_STRG|STRG_VAL|S_CONV|ENO|CONCAT)\\b",
```

Comment:
is s_conv and concat datatypes? those have to be functions, right?

Reply:
Sorry, my fault. Moved it in https://github.com/Gunders89/vscode-scl/pull/20/commits/59b49c151b49c14c098dee9d5aa274f27bf42aeb